### DISCUSSION TITLE
Add support for TLS configuration in Grafana Webhook

### DISCUSSION THREAD

**Discussion by findmyname666** [grafana/grafana#65609] [Alerting]
Hello,

I need to use mtls for the AWS IoT Core e.g. I want to publish MQTT message over https when an alert is firing.
In such case the webhook needs to support TLS configuration.
Currently it supports only [HTTP basic auth](https://github.com/grafana/grafana/blob/main/pkg/services/ngalert/notifier/channels_config/available_channels.go#L776).
Unfortunately it cannot be used in this case.

I'm talking either about contact points or outgoing webhook under oncall section:

![grafana_webhook](https://user-images.githubusercontent.com/35902428/229094546-fec0e1f9-febe-4b29-90af-a01bdb2ffdd5.png)

FYI - I noticed that alertmanager already supports [this](https://prometheus.io/docs/alerting/latest/configuration/#tls_config).

If you agree that this can be beneficial to multiple users I can take a look on implementation (maybe some pointers would be helpful). Without this I need to use some proxy in the middle :(

BR

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82779. Feel free to continue the discussion around this there. Thank you!
